helm-charts/splunk-otel-collector/templates/daemonset.yaml

```diff
@@ -160,6 +160,11 @@ spec:
           then
               setfacl -n -Rm d:m::rx,m::rx,d:g:{{ $agent.securityContext.runAsGroup | default 999 }}:rx,g:{{ $agent.securityContext.runAsGroup | default 999 }}:rx {{ .Values.logsCollection.journald.directory }};
           fi;
+          {{- end }}
```

Comment:
All the changes in this file will have to have similar changes in deployment-gateway.yaml and deployment-cluster-receiver.yaml

Reply:
I think this can be confusing for users without more documentation or using different values to enable this feature.
Persistent queues can be used in the logs, metrics, and traces pipelines of the agent, cluster receiver and gateway. We need to clearly show what is supported in the values.yaml file or in an md file like [advanced-configuration.md ](https://github.com/signalfx/splunk-otel-collector-chart/blob/main/docs/advanced-configuration.md).

Reply:
I added description what is supported in values.yaml [here](https://github.com/signalfx/splunk-otel-collector-chart/blob/bb3588c3843c12c253410567063c8609f133bb57/helm-charts/splunk-otel-collector/values.yaml#L113)